datastore/ndb/overview/README.md

```diff
@@ -0,0 +1,57 @@
+## NDB Overview Sample
+
+This is a sample app for Google App Engine that exercises the [NDB Python API](https://cloud.google.com/appengine/docs/python/ndb/).
+
+See our other [Google Cloud Platform github
+repos](https://github.com/GoogleCloudPlatform) for sample applications and
+scaffolding for other python frameworks and use cases.
+
+## Run Locally
+1. Install the [Google Cloud SDK](https://cloud.google.com/sdk/), including the [gcloud tool](https://cloud.google.com/sdk/gcloud/), and [gcloud app component](https://cloud.google.com/sdk/gcloud-app).
+2. Setup the gcloud tool.
+
+   ```
+   gcloud components update app
+   gcloud auth login
+   gcloud config set project <your-app-id>
+   ```
+   You don't need a valid app-id to run locally, but will need a valid id to deploy below.
+
+1. Clone this repo.
+
+   ```
+   git clone https://github.com/GoogleCloudPlatform/datastore-pthon-samples.git
```

Comment:
should be `git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git`